otx/cli/tools/eval.py

```diff
@@ -111,7 +123,18 @@ def main():
 
     dataset_class = get_dataset_class(template.task_type)
 
-    dataset = dataset_class(test_subset={"ann_file": args.test_ann_files, "data_root": args.test_data_roots})
+    test_ann_files, test_data_roots = args.test_ann_files, args.test_data_roots
+    if os.path.exists(args.data):
+        with open(args.data, "r", encoding="UTF-8") as stream:
+            data_config = yaml.safe_load(stream)
+        stream.close()
+
+        test_ann_files, test_data_roots = (
+            data_config["data"]["test"]["ann-files"],
+            data_config["data"]["test"]["data-roots"],
+        )
+
+    dataset = dataset_class(test_subset={"ann_file": test_ann_files, "data_root": test_data_roots})
```

Comment:
I have a doubt for this logic. It seems that if a workspace contains data.yaml then otx eval will evaluate the trained algorithms through dataset in data.yaml. However what if a user want to evaluate the trained algorithm to another dataset? In this situation user will give the path through --test-data-roots, but this will be replaced to the default data.yaml's information. Could you check this scenario?

Reply:
In my opinion, just like your opinion, it's good to give commands the most importance. Added a function called `configure_dataset` to `otx/cli/utils/config.py` and modified it to import and use it from `train`, `eval`, and `optimize`. could you please check it?

Reply:
LGTM